graylog2-server/src/main/java/org/graylog2/database/entities/ScopedEntity.java

```diff
@@ -26,11 +27,11 @@
  * Entity base class, which can be used to enforce that each entity implementation
  * has the required id and _scope fields.
  */
-
-public abstract class ScopedEntity {
+public abstract class ScopedEntity implements MongoEntity {
     public static final String FIELD_ID = "id";
     public static final String FIELD_SCOPE = "_scope";
 
+    @Override
```

Comment:
You can remove the `id()` method from this abstract class. Auto-value will pull it in from the `MongoEntity` interface.